tracer/test/Datadog.Trace.Tests/Propagators/DistributedPropagatorTests.cs

```diff
@@ -18,9 +19,10 @@ public class DistributedPropagatorTests
     private const ulong SpanId = 2;
     private const int SamplingPriority = SamplingPriorityValues.UserReject;
     private const string Origin = "origin";
-    private const string RawTraceId = "1";
-    private const string RawSpanId = "2";
-    private const string PropagatedTags = "key1=value1;key2=value2";
+    private const string RawTraceId = "1a";
+    private const string RawSpanId = "2b";
+    private const string PropagatedTags = "_dd.p.key1=value1,_dd.p.key2=value2";
```

Comment:
Note to reviewers: `PropagatedTags` resulted in not tags because they were missing `_dd.p.` prefix